### ISSUE TITLE
Truncate too early on 80 character width terminal

### ISSUE THREAD

**Issue by simonschaufi** [dantleech/what-changed#15]
I have a 80 character terminal in ubuntu and the commit message is truncated way too early. For comparison I added a line below to see how much space there is still available.

```
dantleech/what-changed: 9 updated

  laravel/framework 8299b6dc..4de12917

    [2020-03-26 16:43:40] 3c692967 ThomasTrost ...

> |----------------------------------------------------------------------------|
```

**Comment by dantleech** [comment]
hmm, I guess this is the culprit: https://github.com/dantleech/what-changed/blob/master/lib/Adapter/Symfony/Report/ConsoleReport.php

not sure why I truncate at `<Terminal Width> - 30`

**Comment by simonschaufi** [comment]
Looking at the class, I found a lot more bugs as lots of tags (info, comment) are not closed properly

**Comment by dantleech** [comment]
`</>`? that's valid if that's what you mean?

**Comment by simonschaufi** [comment]
I haven't seen this kind of closing tag in symfony console yet.

In lots of cases I see a line like this in the output: `[2020-04-03 05:57:47] 6f19efdc sebastianbergmann</com...` and as you can see, the closing tag is also truncated. That's what I mean.

**Comment by dantleech** [comment]
yes -- that's because we truncate the string with the formatting tags - maybe there is a good solution for this in Symfony, not sure.

**Comment by simonschaufi** [comment]
Maybe ignore the counting of additional characters while you are in a tag and then only count the characters outside. The main point of this issue is the total length. What i mentioned now is actually another bug.
